src/runtime/index.ts

```diff
@@ -61,6 +61,9 @@ export default class Runtime {
   }
 
   async runTestRunHooks(key: string, name: string): Promise<void> {
+    if (this.options.dryRun) {
+      return
+    }
```

Comment:
We may need this in parallel runtime as well

Reply:
Ahh totally missed that! Pushed changes now that cover it.